cmd/snap-confine/mount-support.c

```diff
@@ -39,6 +39,7 @@
 #include "../libsnap-confine-private/cleanup-funcs.h"
 #include "../libsnap-confine-private/mount-opt.h"
 #include "../libsnap-confine-private/mountinfo.h"
+#include "../libsnap-confine-private/privs.h"
```

Comment:
Why are you including this? I don't see where you added sc_privs_drop().

Reply:
Ah, leftover from earlier iteration. I'll nix it.

Reply:
Odd, I don't see it in my diff. HMM

Reply:
Wow, this is so weird. I honestly don't see it in my tree and the history is fully pushed here. How is this possible?

Reply:
Eh, sorry :-) I was on another branch